src/scss/includes/direction.scss

```diff
@@ -395,8 +368,115 @@ button.direction_shortcut {
     display: none;
   }
 
-
   .autocomplete_suggestions {
     height: 100vh;
   }
+
+  // Step by step mobile view
+  .itinerary_mobile_step_by_step {
+
+    .mobile-roadmap-preview-close {
+      text-align: center;
+      padding: 7px 0 0;
+      margin: 0 0 0 11px;
+      background: $background;
+      width: 32px;
+      height: 32px;
+      border-radius: 50%;
+      box-shadow: 0 1px 4px 0 rgba(12, 12, 14, 0.2), 0 0 2px 0 rgba(12, 12, 14, 0.12)
+    }
+
+    .divider {
+      display: none;
+    }
+
+    .itinerary_roadmap_item_icon {
+      width: 36px;
+      .roadmapIcon {
+        margin: 8px 0;
+      }
+    }
+
+    .mobile-roadmap-preview-steps {
+      display: flex;
+      position: fixed;
+      bottom: 0;
+      padding-bottom: 19px;
+      width: 100vw;
+      height: auto;
+      min-height: 99px;
+      overflow: auto;
+      scroll-snap-type: x mandatory;
+      scroll-padding: 0 35px;
+
+      &.expanded {
+        .itinerary_mobile_step .itinerary_roadmap_instruction {
+          max-height: 30vh;
+          overflow: auto;
+        }
+      }
+
+      .itinerary_mobile_step {
+        flex: 0 0 calc(100vw - 70px);
+        margin: 0 6px;
+        position: static;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        scroll-snap-align: start;
+        border: none;
+        border-radius: 12px;
+        box-shadow: 0 1px 4px 0 rgba(12, 12, 14, 0.2), 0 0 2px 0 rgba(12, 12, 14, 0.12);
+
+        &.past {
+          background-color: $grey-lighter;
+
+          .itinerary_roadmap_instruction:after {
+            background: linear-gradient(#e8e8ed77, $grey-lighter);
+          }
+        }
+
+        &.active {
+          border: 1px solid $action-blue-dark;
+          box-shadow: 0 0 0 4px rgba(128, 172, 255, 0.2);
```

Comment:
The shadow effect gets cut at the top. Probably a problem with the height of the list itself.
![Capture d’écran de 2020-12-03 10-37-01](https://user-images.githubusercontent.com/243653/100991793-b2028780-3553-11eb-948a-ed540a201063.png)